app/controllers/comments_controller.rb

```diff
@@ -1,14 +1,14 @@
 class CommentsController < ApplicationController
 
   def create
-    @comment = Comment.new comment_params  
+    @comment = Comment.new comment_params
     @comment.account_id = current_account.id
```

Comment:
can you send the accound_id in the params itself? if that doesn't require much effort.